### PR TITLE
Migrate "message" table in database

### PR DESCRIPTION
**Pull request by ghost**
## Description of Change

- Created an entity for the message 
- Created repository for the message 
- Generated a migration file for the message entity

## Associated Ticket Link(s)

- https://trello.com/c/etY11lxP

## Screenshots / Screen Recordings

N/A

### CODE REVIEW

File: src/common/entities/messages.entity.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  [EntityRepositoryType]?: MessageRepository;
+
+  @PrimaryKey()
+  id!: number;
+
+  @Property({ fieldName: "content" })
+  content!: string;
+
+  @Property({ fieldName: "attachment_url", nullable: true })
+  attachmentURL?: string;
```
Comment (ishmam-iqbal-sazim) [inline src/common/entities/messages.entity.ts]: [NIT]

```suggestion
  attachmentUrl?: string;
```

File: src/common/entities/messages.entity.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  Rel,
+} from "@mikro-orm/core";
+
+import { MessageRepository } from "@/messages/messages.repository";
+
+import { Classroom } from "./classrooms.entity";
+import { CustomBaseEntity } from "./custom-base.entity";
+import { User } from "./users.entity";
+
+@Entity({ tableName: "messages", repository: () => MessageRepository })
```
Comment (ishmam-iqbal-sazim) [inline src/common/entities/messages.entity.ts]: Let's make this plural


```suggestion
@Entity({ tableName: "messages", repository: () => MessagesRepository })
```

File: src/messages/messages.repository.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { Injectable } from "@nestjs/common";
+
+import { EntityRepository } from "@mikro-orm/postgresql";
+
+import { Message } from "@/common/entities/messages.entity";
+
+@Injectable()
+export class MessageRepository extends EntityRepository<Message> {}
```
Comment (ishmam-iqbal-sazim) [inline src/messages/messages.repository.ts]: Same comment as above


```suggestion
export class MessagesRepository extends EntityRepository<Message> {}
```

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Left some comments

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**